homogen - parallel.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import division
+import numpy as np
+import sys
+import os
+import time
+sys.path.append(os.getcwd() + "\\pvtrace")
+PVTDATA = os.getcwd() + "\\pvtrace\\data"
```

Comment:
This is Window's specific. 

Here you should use `os.path.join` as a platform agnostic way of constructing paths.

But better still this should be configured at the system level and thus be independent of the code that is committed here.